src/styles/main.scss

```
@@ -1,3 +1,54 @@
+@import "./utils/variables";
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-padding;
+  max-width: ($day-size * 7) + ($calendar-padding * 2) + 6;
+
+  @for $index from 28 through 31 {
+    &--month-lenght--#{$index} :nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  @each $name, $number in $day-of-week {
+    &--start-day--#{$name} :first-child {
+      margin-left: #{$number * $day-size + $number};
+    }
+  }
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: $day-size;
+    height: $day-size;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer ;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: all 0.5s;
```

don't use `all` for transition, you should specify the props you that should change
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```

---

And better store transition duration in variable